pyomo/contrib/pynumero/__init__.py

```diff
@@ -0,0 +1,14 @@
+try:
+    import numpy as np
+    numpy_available = True
+except ImportError:
+    numpy_available = False
+
+if numpy_available:
+    from .sparse.intrinsic import *
+else:
+    print("WARNING: Numpy not available. Install numpy before using pynumero")
```

Comment:
You might want to turn this into a logging statement instead of a print statement.